app/controllers/register.js

```diff
@@ -0,0 +1,18 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+
```

Comment:
Might be worth adding the properties here, so it's clear we're expecting them and where we get them from in `saveRegistration()`

i.e.
```
user = '',
passwrd = '',
....
```